supplements/fizbans-treasury-of-dragons/spells.xml

```diff
@@ -0,0 +1,195 @@
+			</div>
+		</description>
+		<setters>
+			<set name="level">5</set>
+			<set name="school">Conjuration</set>
+			<set name="time">1 action</set>
+			<set name="range">60 feet</set>
+			<set name="hasVerbalComponent">true</set>
+			<set name="hasSomaticComponent">true</set>
+			<set name="hasMaterialComponent">true</set>
+			<set name="materialComponent">an object with the image of a dragon engraved on it worth at least 500gp</set>
+			<set name="duration">Concentration, up to 1 hour</set>
+			<set name="isConcentration">true</set>
+		</setters>
+	</element>
+
+	<!-- Summons Statblock -->
+	<element name="Draconic Spirit" type="Information" source="Fizban's Treasury Of Dragons" id="ID_WOTC_FTOD_INFORMATION_SUMMON_DRACONIC_STATBLOCK">
+		<description>
+			<p><em>Large Dragon</em></p>
+			<ul class="unstyled mb">
+			<li><strong>Armor Class</strong> 14 + the level of the spell (natural armor)</li>
+			<li><strong>Hit Points</strong>50 + 10 for each spell level above 5th (the dragon has a number of Hit Dice [d10s] equal to the level of the spell)</li>
+			<li><strong>Speed</strong> 30 ft., fly 60 ft., swim 30ft.</li>
+			</ul>
+			<table style="text-align:center">
+				<thead><tr><td>STR</td><td>DEX</td><td>CON</td><td>INT</td><td>WIS</td><td>CHA</td></tr></thead>
+				<tr><td>19 (+4)</td><td>14 (+2)</td><td>17 (+3)</td><td>10 (+0)</td><td>14 (+2)</td><td>14 (+2)</td></tr>
+			</table>
+			<ul class="unstyled mb">
+				<li><strong>Damage Resistances (Chromatic and Metallic Only)</strong> acid, cold, fire, lightning, poison</li>
+				<li><strong>Damage Resistances (Gem Only)</strong> force, necrotic, psychic, radiant, thunder</li>
+				<li><strong>Condition Immunities</strong> charmed, frightened, poisoned</li>
+				<li><strong>Senses</strong> blindsight 30ft., darkvision 60 ft., passive Perception 12</li>
+				<li><strong>Languages</strong> Draconic, understands the languages you speak</li>
+				<li><strong>Challenge</strong> — 	<strong>Proficiency Bonus</strong> equals your bonus.</li>
+			</ul>
+			<p><strong><em>Shared Resistances.</em></strong> When you summon the dragon, choose one of its damage resistances. You have resistance to the chosen damage type until the spell ends.</p>
+			<h5 class="caption">ACTIONS</h5>
+			<p><strong><em>Multiattack.</em></strong> The dragon makes a number of Rend equal to half this spell’s level (rounded down), and uses Breath Weapon.</p>
+			<p><strong><em>Rend.</em></strong> <em>Melee Weapon Attack:</em> your spell attack modifier to hit, reach 10 ft., one target. <em>Hit:</em> 1d6 + 4 + the spell’s level piercing damage.</p>
+			<p><strong><em>Breath Weapon.</em></strong> The dragon exhales destructive energy in a 30-foot cone. Each creature in that area must make a Dexterity saving throw against your spell save DC. A creature takes 2d6 damage of a type this dragon has resistance to (your choice) on a failed save, or half as much damage on a successful one.</p>
```

Comment:
i need to test this, i believe this is a better way to do stat-blocks... honestly, there was so many iterations i kinda lost which one we decided to use. Last one that @kgirtxd did was probably the best, closest to book stat-block.
```suggestion
			<p style="margin-bottom:-5px"><em>Large Dragon, Neutral</em></p>
			<h4 style="margin-top:5px" />
			<ul class="unstyled">
				<li><strong>Armor Class</strong> 14 + the level of the spell (natural armor)</li>
				<li><strong>Hit Points</strong> 50 + 10 for each spell level above 5th (the dragon has a number of Hit Dice [d10s] equal to the level of the spell)</li>
				<li><strong>Speed</strong> 30 ft., fly 60 ft., swim 30 ft.</li>
			</ul>
			<table style="text-align:center">
				<thead><tr><td>STR</td><td>DEX</td><td>CON</td><td>INT</td><td>WIS</td><td>CHA</td></tr></thead>
				<tr><td>19 (+4)</td><td>14 (+2)</td><td>17 (+3)</td><td>10 (+0)</td><td>14 (+2)</td><td>14 (+2)</td></tr>
			</table>
			<ul class="unstyled">
				<li><strong>Damage Resistances (Chromatic and Metallic Only)</strong> acid, cold, fire, lightning, poison</li>
				<li><strong>Damage Resistances (Gem Only)</strong> force, necrotic, psychic, radiant, thunder</li>
				<li><strong>Condition Immunities</strong> charmed, frightened, poisoned</li>
				<li><strong>Senses</strong> blindsight 30ft., darkvision 60 ft., passive Perception 12</li>
				<li><strong>Languages</strong> Draconic, understands the languages you speak</li>
				<li><strong>Challenge</strong> — <strong>Proficiency Bonus</strong> equals your bonus.</li>
			</ul>
			<h4 style="margin-top:5px; margin-bottom:10px" />
			<ul class="unstyled">
				<li><strong><em>Shared Resistances.</em></strong> When you summon the dragon, choose one of its damage resistances. You have resistance to the chosen damage type until the spell ends.</li>
			</ul>
			<h4 style="margin-top:10px">Actions</h4>
			<ul class="unstyled">
				<li><strong><em>Multiattack.</em></strong> The dragon makes a number of Rend equal to half this spell’s level (rounded down), and uses Breath Weapon.</li>
				<li><strong><em>Rend.</em></strong> <em>Melee Weapon Attack:</em> your spell attack modifier to hit, reach 10 ft., one target. <em>Hit:</em> 1d6 + 4 + the spell’s level piercing damage.</li>
				<li><strong><em>Breath Weapon.</em></strong> The dragon exhales destructive energy in a 30-foot cone. Each creature in that area must make a Dexterity saving throw against your spell save DC. A creature takes 2d6 damage of a type this dragon has resistance to (your choice) on a failed save, or half as much damage on a successful one.</li>
			</ul>
```